experiment/quick-sort/simulation/css/quicksort-styles.css

```diff
@@ -49,23 +49,48 @@
 }
 
 .left {
-   background-color: #0000ff;
+  border-radius: 0%;
+  width: 3.2vw;
+  margin-top: 2.8%;
+  height: 0.35vw;
+  border: 0vw solid grey;
+  background-color: #0000ff;
 }
 
 .right {
-   background-color: #ff0000;
+  background-color: #ff0000;
+  border-radius: 0%;
+  width: 3.2vw;
+  margin-top: 2.8%;
+  height: 0.35vw;
+  border: 0vw solid grey;
 }
 
 .merge {
-   background-color: #cc00ff;
+  border-radius: 0%;
+  width: 3.2vw;
+  margin-top: 2.8%;
+  height: 0.35vw;
+  border: 0vw solid grey;
+  background-color: #cc00ff;
 }
 
 .selected-pivot {
-   background-color: #ddddff;
+  border-radius: 0%;
+  width: 3.2vw;
+  margin-top: 2.8%;
+  height: 0.35vw;
+  border: 0vw solid grey;
+  background-color: #ddddff;
 }
 
 .swapping {
-   background-color: #f99b9b;
+  border-radius: 0%;
+  width: 3.2vw;
+  margin-top: 2.8%;
+  height: 0.35vw;
+  border: 0vw solid grey;
+  background-color: #f99b9b;
```

Comment:
in all the above classes, border-radius,width, margin-top, height, border are same, only the bg color is different. Why not create a common class for such things and another for just the background colours? Avoid redundant code.